GitUI/CommitInfo/CommitInfo.Designer.cs

```diff
@@ -171,23 +177,69 @@ private void InitializeComponent()
             this.addNoteToolStripMenuItem.Text = "Add notes";
             this.addNoteToolStripMenuItem.Click += new System.EventHandler(this.addNoteToolStripMenuItem_Click);
             // 
+            // tlpnlRevisionHeader
+            // 
+            this.tlpnlRevisionHeader.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
+            | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));
+            this.tlpnlRevisionHeader.AutoSize = true;
+            this.tlpnlRevisionHeader.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.tlpnlRevisionHeader.BackColor = System.Drawing.SystemColors.ControlLight;
+            this.tlpnlRevisionHeader.ColumnCount = 3;
+            this.tlpnlRevisionHeader.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle(System.Windows.Forms.SizeType.Percent, 100F));
+            this.tlpnlRevisionHeader.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.tlpnlRevisionHeader.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
```

Comment:
this doesn't work properly on linux
![image](https://user-images.githubusercontent.com/4403806/30769369-2c2a6f60-a05a-11e7-86a5-4abfd30ce4bf.png)

i'll try to help with this

Reply:
Add a method to CommitInfo class

```
        private void ApplyLayout()
        {
            if (!EnvUtils.IsMonoRuntime())
            {
                return;
            }

            tlpnlRevisionHeader.ColumnStyles[1].SizeType = SizeType.Absolute;
            tlpnlRevisionHeader.ColumnStyles[1].Width = 38;
            tlpnlRevisionHeader.ColumnStyles[2].SizeType = SizeType.Absolute;
            tlpnlRevisionHeader.ColumnStyles[2].Width = 38;
        }
```

and call it from the constructor.

This will force the size for the columns, they won't be dynamic, but it still provides a reasonable UI/UX.

Reply:
done